RubberduckTests/Grammar/VBAParserTests.cs

```diff
@@ -0,0 +1,142 @@
+        }
+
+        [TestMethod]
+        public void TestModuleHeader()
+        {
+            string code = @"VERSION 1.0 CLASS";
+            string expectedTree = @"
+(startRule
+    (module endOfStatement
+        (moduleHeader VERSION 1.0 CLASS)
+        endOfStatement
+        endOfStatement
+        endOfStatement
+        endOfStatement
+        endOfStatement)
+<EOF>)";
+            assertTree(code, expectedTree);
+        }
+
+        [TestMethod]
+        public void TestModuleConfig()
+        {
+            string code = @"
+BEGIN
+  MultiUse = -1  'True
+END";
+            string expectedTree = @"
+(startRule
+    (module
+        (endOfStatement
+            (endOfLine \r\n))
+        (moduleConfig BEGIN
+            (endOfStatement
+                (endOfLine \r\n ))
+            (moduleConfigElement
+                (ambiguousIdentifier MultiUse) = 
+                (literal -1)
+                (endOfStatement
+                    (endOfLine
+                        (comment 'True))
+                    (endOfLine \r\n)))
+        END)
+        endOfStatement
+        endOfStatement
+        endOfStatement
+        endOfStatement)
+<EOF>)";
+            assertTree(code, expectedTree);
+        }
+
+        [TestMethod]
+        public void TestDictionaryCallLineContinuation()
+        {
+            string code = @"
+Public Sub Test()
+    Set result = myObj _
+    ! _
+    call
+End Sub";
+            string expectedTree = @"
+(startRule
+    (module
+        (endOfStatement
+            (endOfLine \r\n))
+        endOfStatement
+        endOfStatement
+        endOfStatement
+        (moduleBody
+            (moduleBodyElement
+                (subStmt
+                    (visibility Public)
+                    Sub
+                    (ambiguousIdentifier Test)
+                    (argList ( ))
+                    (endOfStatement
+                        (endOfLine \r\n ))
+                    (block
+                        (blockStmt
+                            (setStmt
+                            Set
+                            (implicitCallStmt_InStmt
+                                (iCS_S_VariableOrProcedureCall
+                                    (ambiguousIdentifier result))) =
+                            (valueStmt 
+                                (implicitCallStmt_InStmt
+                                    (iCS_S_VariableOrProcedureCall
+                                        (ambiguousIdentifier myObj) _\r\n
+                                        (dictionaryCallStmt ! _\r\n
+                                            (ambiguousIdentifier
+                                            (ambiguousKeyword call))))))))
+                    (endOfStatement (endOfLine \r\n)))
+                    End Sub))
+                endOfStatement)
+            endOfStatement)
+<EOF>)";
+            assertTree(code, expectedTree);
+        }
+
+        private void assertTree(string code, string expectedTree)
```

Comment:
IMO asserting the string representation of the parse tree makes the grammar tests a bit harder to write than necessary; given we're working directly with ANTLR objects here, perhaps an XPath query on the parse tree could simplify the expectations and avoid having to transcribe the whole expected string representation? Just an idea.